facefusion/processors/frame/modules/face_swapper.py

```diff
@@ -192,9 +202,10 @@ def prepare_source_embedding(source_face : Face) -> Embedding:
 	return source_embedding
 
 
-def prepare_crop_frame(crop_frame : Frame) -> Frame:
-	crop_frame = crop_frame / 255.0
-	crop_frame = crop_frame[:, :, ::-1].transpose(2, 0, 1)
+def prepare_crop_frame(crop_frame : Frame, mean : List[float], standard_deviation : List[float]) -> Frame:
+	crop_frame = crop_frame[:, :, ::-1] / 255.0
```

Comment:
Change code to this, no need for passing via method.
```
model_mean = get_options('model').get('mean')
model_deviation = get_options('model').get('deviation')

crop_frame = (crop_frame - model_mean) / model_deviation
```

Reply:
Done